applications/FluidDynamicsApplication/custom_elements/vms.h

```diff
@@ -100,7 +100,7 @@ namespace Kratos
  */
 template< unsigned int TDim,
           unsigned int TNumNodes = TDim + 1 >
-class VMS : public Element
+class KRATOS_API(FLUID_DYNAMICS_APPLICATION) VMS : public Element
```

Comment:
Fine with the change. Note however that this is a very old element that will be eventually deprecated. My advise is to use the `qsvms` which implements the same formulation with current design standards.

Reply:
Understood.